__tests__/calculations.test.ts

```diff
@@ -17,6 +17,8 @@ describe("DateTime calculations", () => {
 
     expect(utils.isValid(date)).toBeTruthy();
     expect(utils.isValid(invalidDate)).toBeFalsy();
+    expect(utils.isValid(undefined)).toBeTruthy();
```

Comment:
this looks a bit weird to me, but now it's at least consistent trough all libraries